### PR TITLE
Alien Alloys

### PR DESCRIPTION
**Pull request by Core0verload**
This PR adds an "alien tech" RnD branch, hidden and level 0 at roundstart. You can research it by DAing alien items. It costs even more than syndie tech when sold at cargo.

Alien Tech level 1 allows you to produce alien alloy sheets. They cost 4000 metal and 4000 plasma per sheet and are used to build fancy alien-styled tables, floors, beds, lockers, airlocks and walls.

### CODE REVIEW

**Comment by Aloraydrel** [comment]
ayy

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 1]
```diff
@@ -589,17 +596,88 @@ Congratulations! You are now trained for xenobiology research!"}
 	name = "resting contraption"
 	desc = "This looks similar to contraptions from earth. Could aliens be stealing our technology?"
 	icon = 'icons/obj/abductor.dmi'
+	buildstacktype = /obj/item/stack/sheet/mineral/abductor
 	icon_state = "bed"
 
+/obj/structure/table_frame/abductor
+	name = "alien table frame"
+	desc = "A strudy table frame made from alien alloy."
```
Comment (Xhuis) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: sturdy*

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 2]
```diff
@@ -311,4 +311,37 @@ var/global/list/datum/stack_recipe/snow_recipes = list ( \
...
+	force = 5
+	throwforce = 5
+	w_class = 3
+	throw_speed = 1
+	throw_range = 3
+	origin_tech = "materials=6;abductor=1"
+	sheettype = "abductor"
+
+var/global/list/datum/stack_recipe/abductor_recipes = list ( \
+/*	new/datum/stack_recipe("alien chair", /obj/structure/chair, one_per_turf = 1, on_floor = 1), \*/
```
Comment (Iamgoofball) [inline code/game/objects/items/stacks/sheets/mineral.dm]: /*

File: code/modules/research/research.dm [hunk 3]
```diff
@@ -220,12 +220,19 @@ research holder datum.
...
+
 /datum/tech/syndicate
 	name = "Illegal Technologies Research"
 	desc = "The study of technologies that violate Nanotrassen regulations."
 	id = "syndicate"
 	rare = 4
 
+/datum/tech/abductor
+	name = "Alien Technologies Research"
+	desc = "The study of technologies used by advanced alien race known as Abductors."
```
Comment (Xhuis) [inline code/modules/research/research.dm]: the advanced*

File: code/game/machinery/doors/airlock_types.dm [hunk 4]
```diff
@@ -299,11 +299,14 @@
 
 /obj/machinery/door/airlock/abductor
 	name = "alien airlock"
+	desc = "With humanity's current technological level, it could take years to hack this advanced airlock... or maybe we should give screwdriver a try?"
```
Comment (Xhuis) [inline code/game/machinery/doors/airlock_types.dm]: the screwdriver*

**Comment by Xhuis** [comment]
Grammar nazi out

File: code/modules/research/designs.dm [hunk 5]
```diff
@@ -497,3 +497,14 @@ datum/design/diagnostic_hud_night
 	materials = list(MAT_METAL = 1000, MAT_GLASS = 500, MAT_PLASMA = 1500, MAT_URANIUM = 200)
 	build_path = /obj/item/weapon/weldingtool/experimental
 	category = list("Equipment")
+
+
+/datum/design/alienalloy
+	name = "Alien Alloy"
+	desc = "A sheet of reverse-engineered alien alloy."
+	id = "alienalloy"
+	req_tech = list("abductor" = 1, "materials" = 7, "plasmatech" = 2)
```
Comment (PKPenguin321) [inline code/modules/research/designs.dm]: >"materials" = 7

File: code/game/machinery/doors/airlock_types.dm (hunk 4 above)
Comment (PKPenguin321) [inline code/game/machinery/doors/airlock_types.dm]: a screwdriver**

**Comment by PKPenguin321** [comment]
:+1:

**Comment by MMMiracles** [comment]
You may want to instruct your men to exercise restraint when using explosives, commander.

File: code/modules/research/designs.dm (hunk 5 above)
Comment (Core0verload) [inline code/modules/research/designs.dm]: what?

**Comment by Zilenan91** [comment]
Isn't that a lot of materials just for cosmetic furniture

**Comment by Core0verload** [comment]
Well, it's advanced alien materials, they must be costly. And with gooftier upgrades, materials are cheap anyway.

**Comment by KorPhaeron** [comment]
I think it should probably be a bit cheaper, but cool stuff

**Comment by xxalpha** [comment]
Don't make it cheaper, it's fine as is.

**Comment by Steelpoint** [comment]
So when do we get to add Advance Plasma Guns based off of Alien Alloys?

We Xcom now.

File: code/modules/research/designs.dm (hunk 5 above)
Comment (PKPenguin321) [inline code/modules/research/designs.dm]: i think 7 is generally the max for mats research but i dont think 8 actually does anything so nevermind
so actually nothing i guess

**Comment by GunHog** [comment]
This is cooler than cool :+1: The sheets seem a bit expensive, considering the ORM nerf coming down the pipe, but still, it will make for some seriously amazing looking forts.

**Comment by xxalpha** [comment]
Something needs to be expensive, you RND folks take too much for granted.

**Comment by KorPhaeron** [comment]
Yeah, the guns and mechs should be expensive, not the purely cosmetic stuff nobody will make after the first week in the game

**Comment by duncathan** [comment]
I'm sure we can always reduce the cost of the cosmetics if need be

**Pull request merged**